jrnl/upgrade.py

```diff
@@ -11,10 +11,22 @@
 def backup(filename, binary=False):
     print(f"  Created a backup at {filename}.backup", file=sys.stderr)
     filename = os.path.expanduser(os.path.expandvars(filename))
-    with open(filename, "rb" if binary else "r") as original:
-        contents = original.read()
-    with open(filename + ".backup", "wb" if binary else "w") as backup:
-        backup.write(contents)
+
+    try:
+        with open(filename, "rb" if binary else "r") as original:
+            contents = original.read()
+
+        with open(filename + ".backup", "wb" if binary else "w") as backup:
+            backup.write(contents)
+    except FileNotFoundError:
+        print(f"\nError: {filename} does not exist.")
+        try:
+            cont = util.yesno(f"\nCreate {filename}?", default=False)
```

Comment:
I think we want to skip adding the journal to the updated config, rather than creating the file. Creating the file could lead to some awkward experiences with encrypted journals, and possibly break synced files by accidentally overwriting, so it's best to skip.